src/main/java/hw6/core/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,209 @@
+package hw6.core.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+import com.epam.jdi.light.ui.html.elements.common.*;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+
+import hw6.core.entities.datafromjsonfile.*;
+
+import org.openqa.selenium.WebElement;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class MetalsAndColorsPage extends WebPage {
```

Comment:
Where is MetalsAndColors Form<Entity> ? 
Take a look at Login form and Entity Driving Testing approach from the lecture.
My expectations is: Class -> Page -> Form::submit(entity)